common/utils/src/main/scala/org/apache/spark/util/MavenUtils.scala

```diff
@@ -215,7 +215,7 @@ private[spark] object MavenUtils extends Logging {
         if (artifactInfo.getExt == "jar") {
           true
         } else {
-          logInfo(s"Skipping non-jar dependency ${artifactInfo.getId}")
+          logInfo(log"Skipping non-jar dependency ${MDC(LogKeys.ARTIFACT_ID, artifactInfo.getId)}")
```

Comment:
let's drop the changes in this one to avoid adding new log keys.
We can use comments to disable the scala style check.